.expeditor/end_to_end.pipeline.yml

```diff
@@ -255,4 +255,13 @@ steps:
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
             - HAB_BLDR_URL=https://bldr.acceptance.habitat.sh
-  
+
+  - wait
```

Comment:
Education for myself and our fellow readers: the intent of how we release is that _everything_ that passes e2e testing makes it to acceptance.  It's the channels downstream of that that require a human to make a decision on when to promote.  Correct?

Reply:
Yup